static/css/inline-selection.scss

```diff
@@ -253,6 +253,57 @@ $overlay-z-index: 900000;
   z-index: $overlay-z-index;
 }
 
+.pageshot-select-mode-background {
```

Comment:
It might be, I'm not sure – it could be a problem if the reset was on the root, and the site stylesheet has a specific selector that is considered more specific than reset one.  You might also have to do `& * {@extend .pageshot-reset;}` to get the reset to apply to children.  There's some examples already in that file.